models/dex/dex_trades.sql

```diff
@@ -31,8 +31,34 @@ FROM
                 ,evt_index
                 ,unique_trade_id
         FROM {{ ref('uniswap_trades') }}
-        /*
+        
         UNION
-        <add future protocols here>
-        */
+        
+        SELECT
+                blockchain
+                ,project
+                ,version
+                ,block_date
+                ,block_time
+                ,token_bought_symbol
```

Comment:
Nit: I am not fond of the preceding commas here on line breaks. I see this is how it was and that this was merely copy-pasted, but maybe we can break this tread before is spreads. 

cf - https://github.com/duneanalytics/spellbook/pull/1588